README.md

```diff
@@ -6,6 +6,139 @@
 
 Part of the [OpenTOSCA Ecosystem](http://www.opentosca.org)
 
+## User Guide
+
+### Test Environment Connection
```

Comment:
Why "Test Environment Connection"?
Why not "Create client and retrieve applications"? Also make sure you show what the return types of the actions are -> `List<Application>` in this case.